editoast/src/views/mod.rs

```diff
@@ -357,6 +380,12 @@ pub enum AuthorizationError {
     #[error("Forbidden — user has insufficient privileges")]
     #[editoast_error(status = 403)]
     Forbidden,
+    #[error("Forbidden — user must be an admin to impersonate")]
+    #[editoast_error(status = 403)]
+    ForbiddenImpersonate,
```

Comment:
```suggestion
    ForbiddenImpersonation,
```

but I think the `Forbidden` variant semantics already covers this case

Reply:
I prefer to have a different error so that we can easily identify that the problem comes from the impersonalization feature. Many things can cause a simple “forbidden” error.

Reply:
Ok fair